health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/editVillagePopulationPopUP.js

```
@@ -0,0 +1,67 @@
+import React from 'react';
+import { useTranslation } from "react-i18next";
+import { PopUp, Button, Card, LabelFieldPair, FieldV1, Divider, TextInput } from '@egovernments/digit-ui-components';
+
+const EditVillagePopulationPopUp = ({ onClose }) => {
+  const { t } = useTranslation();
+
+  return (
+    <PopUp
+      onClose={onClose}
+      heading={t(`HCM_MICROPLAN_EDIT_POPULATION_LABEL`)}
+      children={[
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)